cmd/run.go

```diff
@@ -367,20 +367,25 @@ func (r *Runner) handleSingleHistory(ctx context.Context, client client.Client,
 		TaskQueue:      feature.TaskQueue,
 		FeatureStarted: r.createTime,
 	}
+	storage := history.Storage{Dir: filepath.Join(r.rootDir, "features", feature.Dir, "history"), Lang: r.config.Lang}
+	// Load all histories from storage to validate against
+	existingSet, err := storage.Load()
+	if err != nil {
+		return err
+	}
+	if !r.config.GenerateHistory && len(existingSet.ByVersion) == 0 {
```

Comment:
Technically:

```suggestion
	if !r.config.GenerateHistory && (r.config.DisableHistoryCheck || len(existingSet.ByVersion) == 0) {
```

But this is probably fine too

Reply:
Yeah, may as well